src/components/TargetDetails/parseUtils.tsx

```diff
@@ -118,13 +118,27 @@ export const parseContent = (text: string | undefined) => {
   });
 };
 
-export const parseExamples = (example: string) => {
-  const lines = splitByNewLine(example);
+export const parseExamples = (example: string | string[]) => {
+  let lines: string[] | undefined = [];
+  if (typeof example === 'object') {
```

Comment:
This is too broad a check. I would replace it with the following, since we don't accidentally want to call `.forEach` on a JSON object. Based on [this](https://stackoverflow.com/questions/767486/how-do-you-check-if-a-variable-is-an-array-in-javascript) Stack Overflow thread, we should update this line like so:
```suggestion
  if (example.prototype === Array) {
```

Reply:
Well, just kidding — that doesn't work, because TS doesn't know what a prototype is? That's kinda silly. I'm settling for `Array.isArray(example)` instead.